src/docs/markdown/caddyfile/directives/tls.md

```diff
@@ -185,6 +187,34 @@ Obtains certificates from an internal certificate authority.
 
 
 
+
+### Certificate Managers
+
+Certificate manager modules are distinct from issuer modules in that use of manager modules implies that an external tool or service is managing the certificate, whereas an issuer module implies that Caddy itself is managing the certificate. (Issuer modules take a Certificate Signing Request (CSR) as input, but certificate manager modules take a TLS ClientHello as input.)
+
+These manager modules come standard with the `tls` directive:
+
+#### tailscale
+
+Get certificates from a locally-running [Tailscale](https://tailscale.com) instance. [HTTPS must be enabled in your Tailscale account](https://tailscale.com/kb/1153/enabling-https/) (or your open source [Headscale server](https://github.com/juanfont/headscale)); and the Caddy process must either be running as root, or you must configure `tailscaled` to give your Caddy user [permission to fetch certificates](https://github.com/caddyserver/caddy/pull/4541#issuecomment-1021568348).
+
+```caddy
+... tailscale
+```
+
+
+#### http
+
+Get certificates by making an HTTP(S) request. The response must have a 200 status code and the body must contain a PEM chain including the full certificate (with intermediates) as well as the private key.
+
+```caddy
+... http <url>
+```
+
+- **url** <span id="url"/> is the fully-qualified URL to which to make the request. It is strongly advised that this be a loopback endpoint for performance reasons.
```

Comment:
I think "loopback" might be confusing, I think it'd be more useful to say "should have as low latency as possible" or something along those lines